### PR TITLE
added simple main.tex file with 'hello world' as content

### PR DESCRIPTION
**Pull request by bjornnorgaard**

### CODE REVIEW

**Review by northguard91 — COMMENTED** [review]
Is there an endline?

**Comment by bjornnorgaard** [comment]
yes

File: main.tex [hunk 1]
```diff
@@ -0,0 +1,7 @@
+\documentclass{article}
+
+\begin{document}
+
+Hello World!
```
Comment (northguard91) [inline main.tex]: indent line

**Review by northguard91 — CHANGES_REQUESTED** [review]

**Review by northguard91 — APPROVED** [review]
great!

**Pull request merged**